src/test/java/uk/gov/hmcts/reform/notificationservice/service/AuthServiceTest.java

```diff
@@ -61,9 +61,9 @@ void should_throw_when_invalid_token_received() {
         willThrow(new InvalidTokenException(invalidToken)).given(validator).getServiceName(anyString());
 
         // when
-        InvalidTokenException exception = catchThrowableOfType(
```

Comment:
is `InvalidTokenException` still needed?

Reply:
`InvalidTokenException` is library exception. From what I see this PR introduces in house exception usage instead straight from the library

Reply:
if so, then yeah, we can just 'translate' `InvalidTokenException` to http response